docs/src/componentDocs/Spacer/markdown/SpacerExamples.mdx

```diff
@@ -0,0 +1,15 @@
+import { SpacerWithFlex } from '../examples/SpacerWithFlex';
+import { SpacerWithPixel } from '../examples/SpacerWithPixel';
+
+# Spacer with Flex
+
+The `<spacer>` adds space between elements. It accepts a flex prop that determines how much relative space to consume.
```

Comment:
```suggestion
The `<Spacer>` adds space between elements. It accepts a `flex` prop that determines how much relative space to consume.
```

Reply:
verbiage updated to match NG doc description